tools/dashboard/README.md

```diff
@@ -0,0 +1,17 @@
+# README
+
+This folder contains a list of dashboards for every benchmark in
+PerfKitBenchmarker. The dashboards can be uploaded and used by PerfKitExplorer.
+
+See:
+
+`https://github.com/GoogleCloudPlatform/PerfKitExplorer` for How to setup
+PerfKitExplorer.
+
+## Setup dashboards
```

Comment:
I dont see how this describes what the user needs to do to deploy the dashboards to the explorer.   The sed commands wont really do anything as-is.